### ISSUE TITLE
`cond-block` does not work when compiled by `raco setup`

### ISSUE THREAD

**Issue by jsmaniac** [racket/scribble#37]
### What version of Racket are you using?
6.4.0.8

### What program did you run?

```
raco pkg install https://github.com/jsmaniac/scribble-enhanced.git#e4f18d1
```

`cond-block` does not work when compiled by `raco setup`. I created a new package and added the following file in `scribllings`, and pushed it to a separate branch: https://github.com/jsmaniac/scribble-enhanced/blob/e4f18d127007d5b6bd3b198008becb498e784180/scribblings/scribble-enhanced.scrbl

```
#lang scribble/manual
@(require scribble/core
          scriblib/render-cond)
@(cond-block
  [else (paragraph (style #f (list))
                   "foo")])
```

When installed it gives this error:
```
> raco pkg install https://github.com/jsmaniac/scribble-enhanced.git#e4f18d1
…
match: no matching clause for (traverse-block #<procedure:.../render-cond.rkt:38:12>)
  context...:
   /nix/store/kkxy912zqmz950vc4kdgsq39rfvgpz29-racket-6.4.0.8/share/racket/collects/racket/match/runtime.rkt:23:0: match:error
   /nix/store/kkxy912zqmz950vc4kdgsq39rfvgpz29-racket-6.4.0.8/share/racket/pkgs/scribble-lib/scribble/contract-render.rkt:58:0: r-part
   /nix/store/kkxy912zqmz950vc4kdgsq39rfvgpz29-racket-6.4.0.8/share/racket/pkgs/scribble-lib/scribble/contract-render.rkt:14:4: render method in .../contract-render.rkt:12:2
   /nix/store/kkxy912zqmz950vc4kdgsq39rfvgpz29-racket-6.4.0.8/share/racket/pkgs/racket-index/setup/scribble.rkt:1418:3
   .../parallel-do.rkt:390:17
   /nix/store/kkxy912zqmz950vc4kdgsq39rfvgpz29-racket-6.4.0.8/share/racket/collects/setup/../racket/private/more-scheme.rkt:261:28
   /nix/store/kkxy912zqmz950vc4kdgsq39rfvgpz29-racket-6.4.0.8/share/racket/collects/setup/parallel-do.rkt:435:20: loop

  context...:
   /nix/store/kkxy912zqmz950vc4kdgsq39rfvgpz29-racket-6.4.0.8/share/racket/collects/setup/parallel-do.rkt:334:4: work-done method in list-queue%
   /nix/store/kkxy912zqmz950vc4kdgsq39rfvgpz29-racket-6.4.0.8/share/racket/collects/setup/parallel-do.rkt:284:17
   /nix/store/kkxy912zqmz950vc4kdgsq39rfvgpz29-racket-6.4.0.8/share/racket/collects/setup/parallel-do.rkt:238:4
   /nix/store/kkxy912zqmz950vc4kdgsq39rfvgpz29-racket-6.4.0.8/share/racket/pkgs/racket-index/setup/scribble.rkt:431:2: make-loop
   /nix/store/kkxy912zqmz950vc4kdgsq39rfvgpz29-racket-6.4.0.8/share/racket/pkgs/racket-index/setup/scribble.rkt:138:0: setup-scribblings
   /nix/store/kkxy912zqmz950vc4kdgsq39rfvgpz29-racket-6.4.0.8/share/racket/collects/setup/setup-core.rkt:70:0: setup-core
   /nix/store/kkxy912zqmz950vc4kdgsq39rfvgpz29-racket-6.4.0.8/share/racket/collects/setup/setup.rkt:65:3
   /nix/store/kkxy912zqmz950vc4kdgsq39rfvgpz29-racket-6.4.0.8/share/racket/collects/pkg/main.rkt:17:0: setup
   (submod /nix/store/kkxy912zqmz950vc4kdgsq39rfvgpz29-racket-6.4.0.8/share/racket/collects/pkg/main.rkt main): [running body]
   /nix/store/kkxy912zqmz950vc4kdgsq39rfvgpz29-racket-6.4.0.8/share/racket/collects/pkg/raco.rkt: [traversing imports]
…
```

Here's the whole installation log

```
> raco pkg install https://github.com/jsmaniac/scribble-enhanced.git#e4f18d1
Querying Git references for scribble-enhanced at https://github.com/jsmaniac/scribble-enhanced.git#e4f18d1
Downloading repository https://github.com/jsmaniac/scribble-enhanced.git#e4f18d1
raco setup: version: 6.4.0.8
raco setup: platform: x86_64-linux [3m]
raco setup: installation name: snapshot
raco setup: variants: 3m
raco setup: main collects: /nix/store/kkxy912zqmz950vc4kdgsq39rfvgpz29-racket-6.4.0.8/share/racket/collects
raco setup: collects paths: 
raco setup:   /home/georges/.racket/snapshot/collects
raco setup:   /nix/store/kkxy912zqmz950vc4kdgsq39rfvgpz29-racket-6.4.0.8/share/racket/collects
raco setup: main pkgs: /nix/store/kkxy912zqmz950vc4kdgsq39rfvgpz29-racket-6.4.0.8/share/racket/pkgs
raco setup: pkgs paths: 
raco setup:   /nix/store/kkxy912zqmz950vc4kdgsq39rfvgpz29-racket-6.4.0.8/share/racket/pkgs
raco setup:   /home/georges/.racket/snapshot/pkgs
raco setup: links files: 
raco setup:   /nix/store/kkxy912zqmz950vc4kdgsq39rfvgpz29-racket-6.4.0.8/share/racket/links.rktd
raco setup:   /home/georges/.racket/snapshot/links.rktd
raco setup: main docs: /nix/store/kkxy912zqmz950vc4kdgsq39rfvgpz29-racket-6.4.0.8/share/doc/racket
raco setup: --- updating info-domain tables ---
raco setup: updating: /home/georges/.racket/snapshot/share/info-cache.rktd
raco setup: --- pre-installing collections ---
raco setup: --- installing foreign libraries ---
raco setup: --- installing shared files ---
raco setup: --- compiling collections ---
raco setup: --- parallel build using 4 jobs ---
raco setup: 3 making: <pkgs>/scribble-enhanced
raco setup: 3 making: <pkgs>/scribble-enhanced/scribblings
raco setup: --- creating launchers ---
raco setup: --- installing man pages ---
raco setup: --- building documentation ---
raco setup: 3 skipping: <planet>/cce/scheme.plt/4/1/scribblings/main.scrbl
raco setup: 1 skipping: <planet>/dherman/pprint.plt/4/4/pprint.scrbl
raco setup: 3 running: <pkgs>/scribble-enhanced/scribblings/scribble-enhanced.scrbl
raco setup: WARNING: duplicate tag: (index-entry (mod-path "debug"))
raco setup:  in: /home/georges/.racket/snapshot/pkgs/debug/debug/scribblings/debug.scrbl
raco setup:  in: /home/georges/.racket/snapshot/pkgs/mischief/mischief/scribblings/mischief.scrbl
raco setup: WARNING: duplicate tag: (mod-path "debug")
raco setup:  in: /home/georges/.racket/snapshot/pkgs/debug/debug/scribblings/debug.scrbl
raco setup:  in: /home/georges/.racket/snapshot/pkgs/mischief/mischief/scribblings/mischief.scrbl
raco setup: 3 rendering: <pkgs>/racket-index/scribblings/main/user/local-redirect.scrbl
raco setup: 2 rendering: <pkgs>/racket-index/scribblings/main/user/release.scrbl
raco setup: 1 rendering: <pkgs>/scribble-enhanced/scribblings/scribble-enhanced.scrbl
raco setup: 0 rendering: <pkgs>/racket-index/scribblings/main/user/search.scrbl
match: no matching clause for (traverse-block #<procedure:.../render-cond.rkt:38:12>)
  context...:
   /nix/store/kkxy912zqmz950vc4kdgsq39rfvgpz29-racket-6.4.0.8/share/racket/collects/racket/match/runtime.rkt:23:0: match:error
   /nix/store/kkxy912zqmz950vc4kdgsq39rfvgpz29-racket-6.4.0.8/share/racket/pkgs/scribble-lib/scribble/contract-render.rkt:58:0: r-part
   /nix/store/kkxy912zqmz950vc4kdgsq39rfvgpz29-racket-6.4.0.8/share/racket/pkgs/scribble-lib/scribble/contract-render.rkt:14:4: render method in .../contract-render.rkt:12:2
   /nix/store/kkxy912zqmz950vc4kdgsq39rfvgpz29-racket-6.4.0.8/share/racket/pkgs/racket-index/setup/scribble.rkt:1418:3
   .../parallel-do.rkt:390:17
   /nix/store/kkxy912zqmz950vc4kdgsq39rfvgpz29-racket-6.4.0.8/share/racket/collects/setup/../racket/private/more-scheme.rkt:261:28
   /nix/store/kkxy912zqmz950vc4kdgsq39rfvgpz29-racket-6.4.0.8/share/racket/collects/setup/parallel-do.rkt:435:20: loop

  context...:
   /nix/store/kkxy912zqmz950vc4kdgsq39rfvgpz29-racket-6.4.0.8/share/racket/collects/setup/parallel-do.rkt:334:4: work-done method in list-queue%
   /nix/store/kkxy912zqmz950vc4kdgsq39rfvgpz29-racket-6.4.0.8/share/racket/collects/setup/parallel-do.rkt:284:17
   /nix/store/kkxy912zqmz950vc4kdgsq39rfvgpz29-racket-6.4.0.8/share/racket/collects/setup/parallel-do.rkt:238:4
   /nix/store/kkxy912zqmz950vc4kdgsq39rfvgpz29-racket-6.4.0.8/share/racket/pkgs/racket-index/setup/scribble.rkt:431:2: make-loop
   /nix/store/kkxy912zqmz950vc4kdgsq39rfvgpz29-racket-6.4.0.8/share/racket/pkgs/racket-index/setup/scribble.rkt:138:0: setup-scribblings
   /nix/store/kkxy912zqmz950vc4kdgsq39rfvgpz29-racket-6.4.0.8/share/racket/collects/setup/setup-core.rkt:70:0: setup-core
   /nix/store/kkxy912zqmz950vc4kdgsq39rfvgpz29-racket-6.4.0.8/share/racket/collects/setup/setup.rkt:65:3
   /nix/store/kkxy912zqmz950vc4kdgsq39rfvgpz29-racket-6.4.0.8/share/racket/collects/pkg/main.rkt:17:0: setup
   (submod /nix/store/kkxy912zqmz950vc4kdgsq39rfvgpz29-racket-6.4.0.8/share/racket/collects/pkg/main.rkt main): [running body]
   /nix/store/kkxy912zqmz950vc4kdgsq39rfvgpz29-racket-6.4.0.8/share/racket/collects/pkg/raco.rkt: [traversing imports]
raco setup: 1 rendering: <pkgs>/racket-index/scribblings/main/user/start.scrbl
raco setup: --- installing collections ---
raco setup: --- post-installing collections ---
raco setup: --- summary of errors ---
raco setup: error: during building docs for <pkgs>/scribble-enhanced/scribblings/scribble-enhanced.scrbl
raco setup:   match: no matching clause for (traverse-block #<procedure:.../render-cond.rkt:38:12>)
raco setup:     context...:
raco setup:      /nix/store/kkxy912zqmz950vc4kdgsq39rfvgpz29-racket-6.4.0.8/share/racket/collects/racket/match/runtime.rkt:23:0: match:error
raco setup:      /nix/store/kkxy912zqmz950vc4kdgsq39rfvgpz29-racket-6.4.0.8/share/racket/pkgs/scribble-lib/scribble/contract-render.rkt:58:0: r-part
raco setup:      /nix/store/kkxy912zqmz950vc4kdgsq39rfvgpz29-racket-6.4.0.8/share/racket/pkgs/scribble-lib/scribble/contract-render.rkt:14:4: render method in .../contract-render.rkt:12:2
raco setup:      /nix/store/kkxy912zqmz950vc4kdgsq39rfvgpz29-racket-6.4.0.8/share/racket/pkgs/racket-index/setup/scribble.rkt:1418:3
raco setup:      .../parallel-do.rkt:390:17
raco setup:      /nix/store/kkxy912zqmz950vc4kdgsq39rfvgpz29-racket-6.4.0.8/share/racket/collects/setup/../racket/private/more-scheme.rkt:261:28
raco setup:      /nix/store/kkxy912zqmz950vc4kdgsq39rfvgpz29-racket-6.4.0.8/share/racket/collects/setup/parallel-do.rkt:435:20: loop
raco setup:   
.raco-wrapped pkg install: packages installed, although setup reported errors
```

**Comment by mflatt** [comment]
I think this was fixed by the same repair as for #34. Can you check with a newer build?

**Comment by jsmaniac** [comment]
@mflatt Yes, it's fixed as of the latest nightly (20160407-4bcb657), so I'm closing this. Thanks!
